### ISSUE TITLE
Error on GNOME 3.36

### ISSUE THREAD

**Issue by AlexWayfer** [ifl0w/RandomWallpaperGnome3#81]
Hi.

There is unresolved #72 issue, for GNOME 3.34, but there is already stable GNOME 3.36 (on Arch Linux) and this extension doesn't work at all. :pensive: 

```
Mar 09 00:52:17 Alex-ArchMainPC gnome-shell[1316]: JS ERROR: Extension randomwallpaper@iflow.space: Error: Tried to construct an object without a GType; are you using GObject.registerClass() when inheriting from a GObject type?
                                                   NewWallpaperElement@/home/alex/.local/share/gnome-shell/extensions/randomwallpaper@iflow.space/elements.js:175:3
                                                   RandomWallpaperMenu@/home/alex/.local/share/gnome-shell/extensions/randomwallpaper@iflow.space/randomWallpaperMenu.js:35:27
                                                   enable@/home/alex/.local/share/gnome-shell/extensions/randomwallpaper@iflow.space/extension.js:21:14
                                                   _callExtensionEnable@resource:///org/gnome/shell/ui/extensionSystem.js:160:32
                                                   loadExtension@resource:///org/gnome/shell/ui/extensionSystem.js:311:26
                                                   _loadExtensions/<@resource:///org/gnome/shell/ui/extensionSystem.js:536:18
                                                   collectFromDatadirs@resource:///org/gnome/shell/misc/fileUtils.js:27:17
                                                   _loadExtensions@resource:///org/gnome/shell/ui/extensionSystem.js:515:19
                                                   _enableAllExtensions@resource:///org/gnome/shell/ui/extensionSystem.js:545:18
                                                   _sessionUpdated@resource:///org/gnome/shell/ui/extensionSystem.js:576:18
                                                   init@resource:///org/gnome/shell/ui/extensionSystem.js:54:14
                                                   _initializeUI@resource:///org/gnome/shell/ui/main.js:244:22
                                                   start@resource:///org/gnome/shell/ui/main.js:138:5
```

**Comment by MattSturgeon** [comment]
I can't reproduce this when running with #77, so this should be fixed by that PR. You can clone that branch locally and install it as per the README while we wait for it to be merged.

**Comment by AlexWayfer** [comment]
> I can't reproduce this when running with #77, so this should be fixed by that PR. You can clone that branch locally and install it as per the README while we wait for it to be merged.

Yep, basically the extension runs with this patch, but Settings does not open.

**Comment by MattSturgeon** [comment]
> the extension runs with this patch, but Settings does not open.

Ah, I _can_ reproduce that issue.

For clarity, is the output you posted before from running with or without #77? The only output I can generate is `object.actor is deprecated` which shouldn't be causing issues (yet).

```
Apr 12 09:02:19 mattdesktop gnome-shell[1404]: RWG3 [INFO] >> Main :: Enable extension.
Apr 12 09:02:19 mattdesktop gnome-shell[1404]: Usage of object.actor is deprecated for PanelMenuButton
                                               get@resource:///org/gnome/shell/ui/environment.js:285:29
                                               RandomWallpaperMenu@/home/matt/.local/share/gnome-shell/extensions/randomwallpaper@iflow.space/randomWallpaperMenu.js:32:3
                                               enable@/home/matt/.local/share/gnome-shell/extensions/randomwallpaper@iflow.space/extension.js:21:14
                                               _callExtensionEnable@resource:///org/gnome/shell/ui/extensionSystem.js:165:32
                                               _onEnabledExtensionsChanged/<@resource:///org/gnome/shell/ui/extensionSystem.js:451:18
                                               _onEnabledExtensionsChanged@resource:///org/gnome/shell/ui/extensionSystem.js:450:11
Apr 12 09:02:19 mattdesktop gnome-shell[1404]: Usage of object.actor is deprecated for PanelMenuButton
                                               get@resource:///org/gnome/shell/ui/environment.js:285:29
                                               updatePanelMenuVisibility@/home/matt/.local/share/gnome-shell/extensions/randomwallpaper@iflow.space/randomWallpaperMenu.js:137:4
                                               init@/home/matt/.local/share/gnome-shell/extensions/randomwallpaper@iflow.space/randomWallpaperMenu.js:116:8
                                               enable@/home/matt/.local/share/gnome-shell/extensions/randomwallpaper@iflow.space/extension.js:22:12
                                               _callExtensionEnable@resource:///org/gnome/shell/ui/extensionSystem.js:165:32
                                               _onEnabledExtensionsChanged/<@resource:///org/gnome/shell/ui/extensionSystem.js:451:18
                                               _onEnabledExtensionsChanged@resource:///org/gnome/shell/ui/extensionSystem.js:450:11```

**Comment by firasdib** [comment]
Here is the error I get

```
JS ERROR: Extension randomwallpaper@iflow.space: Error: Tried to construct an object without a GType; are you using GObject.registerClass() when inheriting from a GObject type?
                                                        NewWallpaperElement@/home/firas/.local/share/gnome-shell/extensions/randomwallpaper@iflow.space/elements.js:175:3
                                                        RandomWallpaperMenu@/home/firas/.local/share/gnome-shell/extensions/randomwallpaper@iflow.space/randomWallpaperMenu.js:35:27
                                                        enable@/home/firas/.local/share/gnome-shell/extensions/randomwallpaper@iflow.space/extension.js:21:14
                                                        _callExtensionEnable@resource:///org/gnome/shell/ui/extensionSystem.js:165:32
                                                        loadExtension@resource:///org/gnome/shell/ui/extensionSystem.js:335:26
                                                        callback@resource:///org/gnome/shell/ui/extensionDownloader.js:220:39
                                                        gotExtensionZipFile/<@resource:///org/gnome/shell/ui/extensionDownloader.js:106:13

```

**Comment by ifl0w** [comment]
This should be resolved thanks to PR #77. The updated version will be released soon.
